lib/commands/help.js

```diff
@@ -25,7 +25,7 @@ function to_help(){
 	var color = chalk[ conf.get('color') ] || noop;
 	try{
 		var content = [
-			usage("Usage: ") + " " + util.format("seeli <%s> [options]", color( "command" ) ),
+			usage("Usage: ") + " " + util.format("%s <%s> [options]", path.basename(process.argv[1]), color( "command" ) ),
```

Comment:
You forgot to require path here.

Also, I think this would better be placed in `conf` as `name` and then the default here could be `path.basename(process.argv[1]) || 'seeli`.

it might also be nice to strip file extensions from the basename. If I run it directly `node ./path/command.js` the name shows up as `commmand.js`